django/db/models/base.py

```diff
@@ -322,11 +322,23 @@ def __new__(cls, name, bases, attrs, **kwargs):
         return new_class
 
     def add_to_class(cls, name, value):
+        if name in cls.__dict__:  # not hasattr() to avoid parent attrs
```

Comment:
This leads to somewhat odd behavior. I assume, this is to allow overriding fields from abstract parents. However, maybe a warning is better suited here, since there might be a reason, why 3rd party overrides an existing attribute.
I think it's a good idea to warn people, but I wouldn't go as far as to actively prevent people from doing so.